libs/enigo/src/win/win_impl.rs

```diff
@@ -48,6 +49,11 @@ fn keybd_event(flags: u32, vk: u16, scan: u16) -> DWORD {
                     GetWindowThreadProcessId(GetForegroundWindow(), std::ptr::null_mut());
                 LAYOUT = GetKeyboardLayout(current_window_thread_id);
             }
+            if winapi::shared::minwindef::LOWORD(LAYOUT as usize as u32) == 0x0412 {
+                if vk == winapi::um::winuser::VK_RMENU as u16 {
+                    vk = winapi::um::winuser::VK_HANGUL as u16;
+                }
+            }
```

Comment:
check `vk == winapi::um::winuser::VK_RMEN` please, because it is cheaper than  LOWORD
``` 
if vk == winapi::um::winuser::VK_RMENU as u16 && winapi::shared::minwindef::LOWORD(LAYOUT as usize 
                  vk = winapi::um::winuser::VK_HANGUL as u16;
            }
```

Reply:
I modified it a little. 
`            if (LAYOUT as u16 == 0x0412) && (vk == winapi::um::winuser::VK_RMENU as u16) { 
                vk = winapi::um::winuser::VK_HANGUL as u16;
            }
`